recipes-core/initscripts/initscripts_1.0.bbappend

```diff
@@ -9,12 +9,8 @@ SRC_URI += " \
 "
 
```

Comment:
Fix `nilrt-mxg` typo in the commit message.

Reply:
Fixed.